src/types/registration/registration.ts

```diff
@@ -0,0 +1,9 @@
+export type RegisterUserData = {
+  name: String;
+  login: String;
+  password: String;
+};
+
+export type RegisterUserFormData = RegisterUserData & {
```

Comment:
Are you sure you will use this type outside of the registration form? Move it to the module level if you don't